### PR TITLE
instructions for creating a build environment were outdated

### PR DESCRIPTION
**Pull request by esc**
As title

### CODE REVIEW

**Comment by 1844144** [comment]
Hi, it worked for me. 

One more thing. In "Building Numba" we should use `git clone git@github.com:numba/numba.git` instead of `git clone git://github.com/numba/numba.git`
image

**Comment by esc** [comment]
> `git clone git@github.com:numba/numba.git`

@1844144 thank you for the review, I have updated the clone url accordingly.

File: docs/source/developer/contributing.rst [hunk 1]
```diff
@@ -128,7 +129,7 @@ Building Numba
 For a convenient development workflow, we recommend you build Numba inside
 its source checkout::
 
-   $ git clone git://github.com/numba/numba.git
+   $ git clone git clone git@github.com:numba/numba.git
```
Comment (1844144) [inline docs/source/developer/contributing.rst:132]:
```suggestion
   $ git clone git@github.com:numba/numba.git
```
Typo

Comment (esc) [reply docs/source/developer/contributing.rst]: Oh yes, good catch, thank you!

**Review by 1844144 — APPROVED** [review]

**Review by sklam — APPROVED** [review]
LGTM

**Pull request merged**